packages/sit-onyx/src/components/OnyxFilterTag/OnyxFilterTag.vue

```diff
@@ -0,0 +1,53 @@
+<script lang="ts" setup>
+import xSmall from "@sit-onyx/icons/x-small.svg?raw";
+import { computed } from "vue";
+import { SKELETON_INJECTED_SYMBOL, useSkeletonContext } from "../../composables/useSkeletonState";
+import { injectI18n } from "../../i18n";
+import OnyxTag from "../OnyxTag/OnyxTag.vue";
+import type { OnyxTagProps } from "../OnyxTag/types";
+
+const props = withDefaults(defineProps<Omit<OnyxTagProps, "" | "color" | "clickable"> & {}>(), {
```

Comment:
What is the pruprose of the empty string and empty object?
```suggestion
const props = withDefaults(defineProps<Omit<OnyxTagProps, "color" | "clickable">>(), {
```